people/people_service.go

```diff
@@ -225,6 +228,14 @@ func (s service) Count() (int, error) {
 	return results[0].Count, nil
 }
 
+func randSeq(n int) string {
```

Comment:
I think this can be done without randomness.  Is this to avoid duplicate variable names in the statement?  If so, perhaps use multiple statements.
Also, this will clash about 1 in 27000 times, which maybe isn't many given our data size.